### PR TITLE
Use Ruby gem library to correctly evaluate vbox version

### PR DESCRIPTION
**Pull request by http-418**
Partial fix for #1049

### CODE REVIEW

File: virtualbox_env.sh [hunk 1]
```diff
@@ -14,9 +14,10 @@ if [[ -z "$VBM" ]]; then
 
     function check_version {
         local MIN_VERSION="4.3"
-        local version=`VBoxManage --version | perl -ne 'm/(\d\.\d)\./; print "$1"'`
+        local MY_VERSION=`VBoxManage --version | perl -ne 'm/(\d\.\d)\./; print "$1"'`
+	VERSION_CHECK=`ruby -e "puts Gem::Version.new($MY_VERSION) > Gem::Version.new($MIN_VERSION)"`
```
Comment (cbaenziger) [inline virtualbox_env.sh]: Why not `local` here?

Comment (http-418) [reply virtualbox_env.sh]: no reason at all

File: virtualbox_env.sh [hunk 2]
```diff
@@ -14,9 +14,10 @@ if [[ -z "$VBM" ]]; then
 
     function check_version {
         local MIN_VERSION="4.3"
-        local version=`VBoxManage --version | perl -ne 'm/(\d\.\d)\./; print "$1"'`
+        local MY_VERSION=`VBoxManage --version | perl -ne 'm/(\d\.\d)\./; print "$1"'`
+	local VERSION_CHECK=`ruby -e "puts Gem::Version.new($MY_VERSION) > Gem::Version.new($MIN_VERSION)"`
```
Comment (cbaenziger) [inline virtualbox_env.sh]: It looks like an indent inconsistency here?

**Comment by cbaenziger** [comment]
Nice change!

**Pull request merged**